.github/workflows/yarn-upgrade.yml

```diff
@@ -38,8 +38,6 @@ jobs:
       - name: Install Tools
         run: |-
           npm -g install lerna npm-check-updates@^9.0.0
-      - name: Build CLI
-        run: cd packages/aws-cdk && ../../scripts/buildup
```

Comment:
integ runner depends on the CLI...is there a reason to do both of these buildups?